.pydevproject

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please remove this file from the PR, thanks!
(Also make sure it's ignored in the .gitignore file)